src/center-service/app/controllers/management_requests_controller.rb

```diff
@@ -99,6 +99,10 @@ def handle_auth_cert_registration
 
     auth_cert_bytes = String.from_java_bytes(req_type.getAuthCert())
 
+    owner = member_id(req_type.getServer())
+    # Auto-approval must be enabled and Security Server owner must be registered on Central Server
+    auto_approve = auto_approve_auth_cert_reg_requests? && !SecurityServerClient.find_by_id(owner).nil?
```

Comment:
Updated variable name from `auto_approve` to `auto_approve_and_owner_exists` that better describes the joint variable. I prefer to use a single variable, because it's simpler to check whether the condition is true later in the function.